package.json

```diff
@@ -1,14 +1,25 @@
 {
-  "name": "chorus",
-  "private": true,
-  "type": "module",
-  "version": "1.22.2",
-  "scripts": {
-    "build": "rollup -c",
-    "watch": "rollup -c -w"
-  },
-  "devDependencies": {
-    "rollup": "3.28.1",
-    "rollup-plugin-chrome-extension": "3.6.12"
-  }
+    "name": "chorus",
+    "private": true,
+    "type": "module",
+    "version": "1.22.2",
+    "scripts": {
+        "build": "rollup -c",
+        "watch": "rollup -c -w",
+        "prepare": "husky"
```

Comment:
@sstephanyy: Mind adding a format script to prettify the files? This way a user can run prettier on their own and not just on a commit.